### ISSUE TITLE
Material Design Is In Maintenance Mode

### ISSUE THREAD

**Issue by oarabiledev** [oarabiledev/material-design#10]
_TL;DR: Material Design Plugin For DroidScript is in maintenance mode, basically it will only receive security/bug updates and updates in-case google changes the JSON Schema for the color definitions._

## What is happening?

After the release of this plugin I realized that the way code is structured will make it hard to update the code-base it is rigid. Furthermore, native controls aren't the new hotness in DroidScript, Enjine-IO is. If i continue to develop on this old soon to be replaced API, I may be the reason why DroidScript developers remain stagnant.

Furthermore it is hard to maintain this library as I do it in my spare time, it is mentally draining and I am doing it while unpaid.

## What next?

I'd urge you to get suited with the new Enjine.IO Framework and build on it, I will be also sailing this ship.

I have already shipped some plugins for this Meta-Framework, Check out [Metro Plugin](https://github.com/oarabiledev/metro).

## What are alternatives?

If you are in maintenance mode for a native app, use MUI controls - baked into Ds, you can also use : 

- Support
- BottomNav
- DsNav
- Lottie
- Animator
- PopUp
